authenticate/handlers/device-enrolled.go

```diff
@@ -0,0 +1,23 @@
+package handlers
+
+import (
+	"html/template"
+	"net/http"
+
+	"github.com/pomerium/pomerium/internal/frontend"
+	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/urlutil"
+)
+
+// DeviceEnrolled displays an HTML page informing the user that they've successfully enrolled a device.
+func DeviceEnrolled() http.Handler {
+	tpl := template.Must(frontend.NewTemplates())
+	type TemplateData struct {
+		DeviceCredentialID string
+	}
+	return httputil.HandlerFunc(func(w http.ResponseWriter, r *http.Request) error {
+		return tpl.ExecuteTemplate(w, "device-enrolled.html", TemplateData{
+			DeviceCredentialID: r.FormValue(urlutil.QueryDeviceCredentialID),
```

Comment:
I might have missed it but it doesn't look like this url's HMAC is checked. It's probably not a huge deal but it could mean this value could be spoofed, and this url could be redirected to from another source besides pomerium.

Reply:
It is not. If I HMAC it we will have a double-HMAC'd URL that will expire almost immediately. I can do this but it won't be useful for what this page was supposed to be since we're generating links which last a very long time. (the redirect URI is a component of the generated link)

Reply:
Understood. I was thinking that the webauthn handler would create a signed url on redirect. I get why that doesn't make sense if the endpoint is not Pomerium (and therefore doesn't want or need the hmac'd overhead). 

OK. Can an attacker craft a url that results in this page displaying an arbitrary crafted device credential id? Do we care? I guess we can accomplish the same thing by providing the user a link back to their user info page (`.pomerium`).